frontend/rolecall/src/app/user/user-editor.component.ts

```diff
@@ -72,7 +72,7 @@ export class UserEditor implements OnInit {
   rolesNamesMap = {
     isAdmin: 'Is Admin',
     // TODO: Spelling error in `choreographer`.
```

Comment:
I believe the TODO will be removed from the other PR, but wouldn't hurt to remove it here as well.

Reply:
Done.